README.md

```diff
@@ -18,6 +18,8 @@ git clone https://github.com/cooklang/cook-in-c.git
 ```
 
 
+
+

```

Comment:
why adding extra spaces ? I would be more in favor of removing the many useless new lines, and maybe setup a markdown linter so we have a clear way to format markdown